srcpkgs/deluge/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'deluge'
 pkgname=deluge
-version=1.3.15
-revision=4
+version=2.0.3
+revision=1
 archs=noarch
 build_style=python2-module
```

Comment:
Deluge now supports Python 3, we should use python3 wherever we can.

build_style=python3-module

Reply:
Already WIP, see discussion.